README.md

```diff
@@ -50,35 +50,69 @@ credentials as well as utility methods to create them and to get Application Def
 
 ## Quickstart
 
+### Using Maven
+
+#### Google Auth Library Bill of Materials
+In order to ensure transitive dependencies and the modules themselves are aligned with each other,
+we rely on the Google Auth Library Bill of Materials. Please add this to your dependency management
+section as follows:
+
+[//]: # ({x-version-update-start:google-auth-library-bom:released})
+```xml
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>com.google.auth</groupId>
+      <artifactId>google-auth-library-bom</artifactId>
+      <version>1.30.1</version>
+      <type>pom</type>
+      <scope>import</scope>
+    </dependency>
+  </dependencies>
+</dependencyManagement>
+```
+[//]: # ({x-version-update-end})
+
+#### Choosing your implementation
+
 If you are using Maven, add this to your pom.xml file (notice that you can replace
 `google-auth-library-oauth2-http` with any of `google-auth-library-credentials` and
 `google-auth-library-appengine`, depending on your application needs):
 
-[//]: # ({x-version-update-start:google-auth-library-oauth2-http:released})
-
 ```xml
 <dependency>
   <groupId>com.google.auth</groupId>
+  <!-- Let the BOM manage the transitive dependencies and module version. -->
+  <!-- Replace with the implementation module that suits your needs -->
   <artifactId>google-auth-library-oauth2-http</artifactId>
-  <version>1.19.0</version>
 </dependency>
 ```
-[//]: # ({x-version-update-end})
-
 
+### Using Gradle
 If you are using Gradle, add this to your dependencies
 
-[//]: # ({x-version-update-start:google-auth-library-oauth2-http:released})
+[//]: # ({x-version-update-start:google-auth-library-bom:released})
 ```Groovy
-implementation 'com.google.auth:google-auth-library-oauth2-http:1.19.0'
+dependencies {
+    // The BOM will manage the module versions and transitive dependencies
+    implementation platform('com.google.auth:google-auth-library-bom:1.30.1')
```

Comment:
can this version be updated in release-please?

Reply:
Yes, I recently fixed the readme updates in https://github.com/googleapis/google-auth-library-java/pull/1616 and confirmed it works in https://github.com/googleapis/google-auth-library-java/pull/1618/files

Reply:
Awesome, thank you!